instance/tests/models/test_openedx_database_mixins.py

```diff
@@ -470,6 +479,9 @@ def test_ansible_settings_mongo(self):
         self.assertIn('FORUM_MONGO_PORT: {0}'.format(MONGODB_SERVER_DEFAULT_PORT), ansible_vars)
         self.assertIn('FORUM_MONGO_DATABASE: {0}'.format(self.instance.forum_database_name), ansible_vars)
 
+        # Unset self.instance to avoid tearDown failures trying to deprovisioning databases that were not created
+        self.instance = None
```

Comment:
Why are these lines needed? I thought Matjaz said that "The methods that deprovision databases already seem to be designed so that they won't fail if the database or user does not exist"

If I go into the shell and I go:
```python
instance = production_instance_factory(name="Production instance", sub_domain="production")
instance.delete()
```
Does that still work without error (no actual provisioning or deprovisioning happens)?